psl/psl-core/src/builtin_connectors/mysql_datamodel_connector.rs

```diff
@@ -55,6 +55,7 @@ const CAPABILITIES: ConnectorCapabilities = enumflags2::make_bitflags!(Connector
     MultiSchema |
 
     FullTextIndex |
+    FullTextSearch |
```

Comment:
Where does this come from?

Reply:
This is small addition from me. For full text search, we have two features `FullTextSearchWithIndex` and `FullTextSearchWithoutIndex`. So, inside of query graphq builder, if we want to check if fts is supported we need to check both. I've added this feature so we could do one check for cases where we don't care about which kind of text search is supported.